lib/mayaUsd/fileio/orphanedNodesManager.cpp

```diff
@@ -260,6 +260,11 @@ void OrphanedNodesManager::clear() { pulledPrims().clear(); }
 
 bool OrphanedNodesManager::empty() const { return pulledPrims().root()->empty(); }
 
+OrphanedNodesManager::Memento OrphanedNodesManager::preserve() const
+{
+    return Memento(deepCopy(pulledPrims()));
+}
+
```

Comment:
The refactor of the code that create a Memento into a public function is necessary to be able to save and restore the orphaned nodes manager when saving and loading a Maya scene.